data/json/snippets/zombie_anatomy.json

```diff
@@ -171,6 +171,72 @@
       "You cut into its abdomen, and discover its utter lack of internal organs, the expected cavity replaced by nothing but more flesh and muscle."
     ]
   },
+  {
+    "type": "snippet",
+    "category": "<zombie_harvest_grabber_general>",
+    "text": [
+      "The sheer remaining tension in each of the muscles of its arms and shoulders makes them unwind with a sonorous thwang each time you cut one of the supporting bone.",
+      "The tremendous growth of its upper body's muscle mass must have rendered the creature almost incapable of efficiently surveying its surroundings.",
+      "You realize the muscles of the zombie's arms are hollow, their marrow replaced by a thick oily black liquid. You wonder if the ridiculous muscle mass surrounding them is the other half of a strange hydraulic system.",
+      "You cut into the arms of the creature and realize that instead of what you would expect from what you know of normal human anatomy, its muscles wind around the bones in what looks like a tight spring..",
```

Comment:
```suggestion
      "You cut into the arms of the creature and realize that instead of what you would expect from what you know of normal human anatomy, its muscles wind around the bones in what looks like a tight spring.",
```